### PR TITLE
Set initial values to null

### PR DESCRIPTION
**Pull request by sirkrisp**
See https://github.com/wellyshen/react-cool-dimensions/issues/674#issuecomment-1235761855

## Checklist

- [x ] Documentation added
- [x ] Tests
- [x ] TypeScript definitions updated
- [ ] Ready to be merged (examples might need update)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,6 @@
 node_modules
 dist
 coverage
-.size-snapshot.json
\ No newline at end of file
+.size-snapshot.json
+
+.idea
```
Comment (wellyshen) [inline .gitignore:9]: @sirkrisp Do we need this?

**Comment by wellyshen** [comment]
@sirkrisp Thank you for the PR bro, it looks really nice 😎. I have left a comment, could you please take a look?

**Comment by sirkrisp** [comment]
Thanks! If you are fine that I upload my .idea folder (I guess commonly done), we can delete that line 👍  

Two important notes: 

1. I haven't updated the example, though, so I think it's important we test that before release. Since you know the example the best maybe you can try?
2. Also I haven't added a test case for that specific "null" case.

**Comment by wellyshen** [comment]
> Thanks! If you are fine that I upload my .idea folder (I guess commonly done), we can delete that line 👍
Can we keep the original `.gitignore` file?

> I haven't updated the example, though, so I think it's important we test that before release. Since you know the example > > the best maybe you can try?
> Also I haven't added a test case for that specific "null" case.
I will take care these things, thanks!

**Comment by sirkrisp** [comment]
> Can we keep the original .gitignore file?

Yes

> I will take care of these things, thanks!

awesome, thanks!

**Comment by wellyshen** [comment]
@sirkrisp Thank you bro, I will release it soon 😎

**Comment by wellyshen** [comment]
@all-contributors please add @sirkrisp for code
